tools/webinstall/install.sh

```diff
@@ -1064,9 +1064,9 @@ install_linux_gnu() {
         need_cmd "$GIT"
         need_cmd "$MAKEPKG"
         need_cmd "$RM"
-
         do_cmd "$GIT clone https://aur.archlinux.org/kraftkit-bin.git /tmp/kraftkit-bin"
-        do_cmd "$MAKEPKG -si /tmp/kraftkit-bin"
+        cd /tmp/kraftkit-bin
+        do_cmd "$MAKEPKG -si"
```

Comment:
I would suggest returning to the previous directory after running the command, as we are deleting it anyways 1 line below.

```suggestion
        cd /tmp/kraftkit-bin
        do_cmd "$MAKEPKG -si"
        cd - 1> /dev/null
```
This works for me on `sh`, please confirm it also works for you